### PR TITLE
Created a Makefile, dockerfiles for the server and the example publish…

### PR DESCRIPTION
**Pull request by razamiDev**
…er and subscriber. Created 3 kubernetes yaml files for deployment

### CODE REVIEW

**Comment by johnbelamaric** [comment]
We already are using `dep` not glide, please use that instead.

**Comment by razamiDev** [comment]
Okay, I will switch it.

File: Godeps/Readme [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This directory tree is generated automatically by godep.
+
+Please do not edit.
+
+See https://github.com/tools/godep for more information.
```
Comment (johnbelamaric) [inline Godeps/Readme]: I think you used the wrong tool. We are using:

https://github.com/golang/dep

File: examples/hello/publisher/main.go [hunk 2]
```diff
@@ -15,19 +15,38 @@ import (
...
 var topic = flag.String("topic", hello.DefaultTopicName, "the topic to publish to")
 
 func main() {
 	flag.Parse()
-	conn, err := grpc.Dial(":8080", grpc.WithInsecure())
+	conn, err := grpc.Dial(*url, grpc.WithInsecure())
 	if err != nil {
 		log.Fatalf("failed to dial to grpc server: %v", err)
 	}
+<<<<<<< HEAD
```
Comment (johnbelamaric) [inline examples/hello/publisher/main.go]: Uh-oh, this file has conflicts still...

Comment (dmacthedestroyer) [reply examples/hello/publisher/main.go]: you're too quick! I noticed this after pushing to Roman's remote... I guess I still have to learn how to use this new merge tool of mine. Should be all fixed now.

**Pull request merged**